test/cppsuite/tests/bounded_cursor_stress.cpp

```diff
@@ -638,44 +640,44 @@ class cursor_bound_01 : public test {
                   key_size, characters_type::ALPHABET);
 
                 int exact = 0;
-                range_cursor->set_key(range_cursor.get(), srch_key.c_str());
-                auto ret = range_cursor->search_near(range_cursor.get(), &exact);
+                bounded_cursor->set_key(bounded_cursor.get(), srch_key.c_str());
+                auto ret = bounded_cursor->search_near(bounded_cursor.get(), &exact);
                 testutil_assert(ret == 0 || ret == WT_NOTFOUND || ret == WT_ROLLBACK);
                 if (ret == WT_ROLLBACK)
                     continue;
 
                 /* Verify the bound search_near result using the normal cursor. */
-                validate_bound_search_near(ret, exact, range_cursor, normal_cursor, srch_key,
-                  bound_pair.first, bound_pair.second);
```

Comment:
I would just use bound_set instead of separating to lower bound and upper bound.

Reply:
Addressed this and other review comments here:
[c0b60a7](https://github.com/wiredtiger/wiredtiger/pull/8223/commits/c0b60a7e679a4b383f5507dd0caf13a8467c4c37)